drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/pom.xml

```diff
@@ -19,6 +19,21 @@
     <java.module.name>org.drools.wb.scenariosimulation.editor.client</java.module.name>
   </properties>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+          <configuration>
+            <excludeFilterFile>${project.basedir}/../../../src/main/spotbugs/spotbugs-exclude.xml</excludeFilterFile>
```

Comment:
look at my conversation with @gitgabrio at https://github.com/kiegroup/drools-wb/pull/1015#discussion_r259234514, the filter can be removed completely.